crawler.js

```diff
@@ -463,9 +497,17 @@ class Crawler {
     return true;
   }
 
+  createPage(url){
+    var id = uuidv4();
+    var today = new Date();
+    var row = {"id": id, "url": url.href, "ts": today}
```

Comment:
Currently, py-wacz would reject the page, if the `ts` doesn't match the timestamp in the WARC/CDX.
Depending on how slow the page loading is, this may or may not always work.

I was thinking we can just omit the `ts` here and let the auto-detection find the best match.
Or, we could alter py-wacz, and use this as a hint, to find the closest match to this ts?

Also, could add the title here as that should be available from `page.title()`